spec/factories.rb

```diff
@@ -0,0 +1,19 @@
+require 'faker'
+
+FactoryBot.define do
```

Comment:
We usually have one factory per file, called user_factory.rb, book_factory.rb

Reply:
Done